test/topojson/test.ml

```diff
@@ -92,6 +98,8 @@ and remove acc k = function
   | (k', _) :: rest when k = k' -> List.rev acc @ rest
   | x :: rest -> remove (x :: acc) k rest
 
+let expected_foreign_members = "arcs"
```

Comment:
Foreign members are expected to be of type `(string * Ezjsonm.value) list`.

Reply:
@patricoferris, I have corrected this. Did I get it right?